.rubocop.yml

```diff
@@ -35,6 +35,10 @@ Metrics/ModuleLength:
 Metrics/ClassLength:
   Max: 200
 
+# Limit blocks to have a length of 50 (only needed for physiqual.gemspec)
+Metrics/BlockLength:
+  Max: 50
+
```

Comment:
Toen ik dat deed toen vond ie in allemaal andere bestanden (specs) ineens allemaal blocks die te lang waren. Ik heb deze nu weggehaald en een rubocop:disable:enable voor Metrics/BlockLength direct in physiqual.gemspec gezet.